store/urls.py

```diff
@@ -25,9 +28,23 @@
     permission_classes=[permissions.AllowAny]
 )
 
+router = SimpleRouter()
+router.register(
+    prefix='listings',
+    viewset=ListingViewSet,
+    basename='listings'
+)
+
 urlpatterns = [
+    path('api/', include(router.urls)),
```

Comment:
what's this for? I don't think it's needed.

Reply:
Removed